lib/IDE/Refactoring.cpp

```diff
@@ -2070,6 +2070,211 @@ bool RefactoringActionExpandTernaryExpr::performChange() {
   return false; //don't abort
 }
 
+bool RefactoringActionConvertIfLetExprToGuardExpr::
+	isApplicable(ResolvedRangeInfo Info, DiagnosticEngine &Diag) {
+
+  if (Info.Kind != RangeKind::SingleStatement
+      && Info.Kind != RangeKind::MultiStatement)
+    return false;
+
+  if (Info.ContainedNodes.empty())
+    return false;
+
+  IfStmt *If = nullptr;
+
+  if (Info.ContainedNodes.size() == 1) {
+    if (auto S = Info.ContainedNodes[0].dyn_cast<Stmt*>()) {
+        If = dyn_cast<IfStmt>(S);
```

Comment:
We should always use 2-spaces indentation. Could you fix other indentation issues as well in this change?

Reply:
Fixed.